### PR TITLE
Create sample type model

### PR DESCRIPTION
**Pull request by gregdingle**
# Description

As part of standardizing sample types, this creates a new model and populates it. 

# Notes

* A following PR will create a controller endpoint.

# Tests

Run migration, see no errors, see data in database
Check validation rules with initial data

### CODE REVIEW

File: spec/factories/sample_types.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+FactoryBot.define do
```
Comment (gregdingle) [inline spec/factories/sample_types.rb]: not sure if i should delete these stub files ... we have many that have not been deleted.

**Comment by jshoe** [comment]
Looks like brakeman has a warning to consider

**Comment by gregdingle** [comment]
> Looks like brakeman has a warning to consider

Yup, I had to learn about ruby regex quirks today.

File: spec/factories/sample_types.rb (hunk 1 above)
Comment (jshoe) [reply spec/factories/sample_types.rb]: probably don't need if not being used yet

File: app/models/sample_type.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# From https://docs.google.com/spreadsheets/d/1_hPkQe5LI0Zw_C0Ls4HVCEDsc_FNNVOaU_aAfoaiZRE/
```
Comment (jshoe) [inline app/models/sample_type.rb]: maybe include a title to search for in case the doc moves?

**Comment by gregdingle** [comment]
Sorry I didn't understand that you had comments coming. I will address them in the next PR coming shortly.

**Pull request merged**